_dev/apps/ui/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue

```diff
@@ -280,6 +281,10 @@ export default defineComponent({
       return this.$t(`tooltip.attributeMapping.${name}`);
     },
     nextStep() {
+      this.mappingAttributes.forEach((attribute) => {
+        this.hasError = attribute.fields.some((value) => value.mapped?.length === 0);
+      });
+      if (this.hasError) return;
```

Comment:
It's not good. We will test all the values ​​one by one and directly change the hasError however if the last one does not have one the field will be false THEN there may be errors before.
You should rather reset the hasError at the beginning (false) then do a hasError = myMethod || hasError to let the possibility to be true